config/op-consumer-for-auth/mapping/queries/op2pd/pass/pass-properties.sparql

```diff
@@ -0,0 +1,18 @@
+CONSTRUCT {
+  ?s ?p ?o .
+}
+WHERE {
+  ?s ?p ?o .
+  FILTER EXISTS {
+    ?s a ?type .
+    VALUES ?type {
+      <http://data.vlaanderen.be/ns/besluit#Bestuurseenheid>
+      <http://www.w3.org/ns/org#Organization>
+    }
+  }
+  FILTER (?p IN (
+    <http://www.w3.org/1999/02/22-rdf-syntax-ns#type>,
+    <http://mu.semte.ch/vocabularies/core/uuid>,
+    <http://purl.org/dc/terms/identifier>
```

Comment:
You can remove this predicate because it's already inserted in the public graph by the `identifier-kbo.rq` and `identifier-ovo.rq` files